agent/installer/cli.go

```diff
@@ -88,15 +88,28 @@ func listSupported() {
 		flags    = 0
 	)
 	w.Init(os.Stdout, minwidth, tabwidth, padding, '\t', flags)
-	defer w.Flush()
-
-	fmt.Fprintf(w, "%s\t%s\t%s\n", "OS", "K8S Version", "BYOH Bundle Name")
-	fmt.Fprintf(w, "%s\t%s\t%s\n", "---", "-----------", "----------------")
+	defer func() {
+		err := w.Flush()
+		if err != nil {
+			klogger.Error(err, "Failed to flush the tabwriter")
+		}
+	}()
```

Comment:
AFAIK defer funcs can't return

Reply:
TIL that they can :) will push a fix, there will be multiple fixes

Reply:
Digging more i realised that none of the functions in cli return but handle error and the caller is not made aware of potential error, we should handle this in a new GH issue as this will change core logic wdyt?
For this fix i will keep it as in line with other steps in this function

Reply:
@dharmjit ☝️

Reply:
Tldr - i think we can keep it as is

I wouldn't worry too much about cli.go right now because it's just used as testing installer changes locally. If we end up using it more or if the installer becomes complex, we can revisit then.